components/ModTopbar.vue

```diff
@@ -1,18 +1,19 @@
 <template>
     <!-- styling will change from px to rm -->
-    <div class="h-[76px] bg-emerald-100">
+    <div class="h-[76px] w-full">
         <div v-if="store.activeScreen === ModerationScreen.dashboard ">
             <p class="text-xl font-bold">TOPBAR MODERATION DASHBOARD</p>
         </div>
-        <div v-else-if="store.activeScreen === ModerationScreen.editSubmission">
-            <p class="text-xl font-bold">TOPBAR REVIEW SUBMISSION</p> 
+        <div class="h-[76px] w-full" v-else-if="store.activeScreen === ModerationScreen.editSubmission">
+           <ModEditSubmissionTopbar id="MattIsAGreatHumanBeing"/>
```

Comment:
I agree that Matt is a great human being! Let's change this id to one that we can test later using `data-testid=""`

Reply:
That id there is a prop at the moment that we are passing but we will add a data test id as well!